css/grids.css

```diff
@@ -0,0 +1,240 @@
+/* Cart Container */
+.grid-2-column {
+	display: grid;
+	grid-template-areas: "g-col-1 g-col-2";
+	height: auto;
+	width: max-content;
+}
+
+.cart-content {
+	grid-area: g-col-1;
+	width: max-content;
+}
+
+.cart-billing {
+	grid-area: g-col-2;
+	border-left: 0.2rem solid var(--cta-color);
+	width: max-content;
+	flex-direction: column;
+	align-items: start;
+}
+
+.cart-billing > * {
+	margin: 0 2rem;
```

Comment:
Use global variables for spacing for properties like margin, padding, and gap to maintain consistency across the website

Reply:
Thanks will make that change once I finish with other assignments